src/components/ContextBar.js

```diff
@@ -93,7 +93,7 @@ export const ContextBar = ({ setDeletingWorkspace, setCloningWorkspace, setShari
       div({ style: contextBarStyles.contextBarContainer }, [
         h(WorkspaceMenuTrigger, { canShare, isOwner, setCloningWorkspace, setSharingWorkspace, setDeletingWorkspace }, [
           h(Clickable, {
-            'aria-label': 'Menu',
+            'aria-label': 'Workspace menu',
```

Comment:
This `aria-label` is actually entirely unused; if you read the definition of `Clickable`, you'll see that when it has a tooltip but no text inside of it, the tooltip is attached via `aria-labelledby`. If you want to change the label to workspace menu, you should change the tooltip.

That said, double checking: this will replace the tab bar, right? Otherwise this needs a different name, since the tab bar we have is already labelled 'workspace menu'.